buster/chatbot.py

```diff
@@ -160,10 +160,7 @@ def get_gpt_response(cls, **completion_kwargs):
 
         except Exception as e:
             # log the error and return a generic response instead.
-            import traceback
-
-            logger.exception("Error connecting to OpenAI API")
-            logging.exception(traceback.format_exc())
+            logger.exception("Error connecting to OpenAI API. See traceback: %s", e)
```

Comment:
The traceback is automatically added to the message. You don't need to append it.

Reply:
nice, you are right, i didn't know that! updated